geometry/spline.cpp

```diff
@@ -52,20 +76,19 @@ void Spline::ReplacePoint(PointD const & pt)
   AddPoint(pt);
 }
 
-bool Spline::IsPrelonging(PointD const & pt)
+bool Spline::IsProlonging(PointD const & pt) const
 {
-  if (m_position.size() < 2)
+  size_t const sz = m_position.size();
+  if (sz < 2)
     return false;
 
   PointD dir = pt - m_position.back();
   if (dir.IsAlmostZero())
     return true;
-
   dir = dir.Normalize();
-  PointD prevDir = m_direction.back().Normalize();
 
-  double const MAX_ANGLE_THRESHOLD = 0.995;
```

Comment:
Let's leave `double constexpr kMaxAngleThreshold = 0.995;` for readability.

Reply:
This constant name is confusing. This is vector-dot-product threshold, which becomes clear one line below without constant.